akka-docs/src/main/paradox/typed/distributed-pub-sub.md

```diff
@@ -0,0 +1,204 @@
+# Distributed Publish Subscribe in Cluster
+
+@@@ note
+For the Akka Classic documentation of this feature see @ref:[Classic Distributed Publish Subscribe](../distributed-pub-sub.md).
+@@@
+
+@@project-info{ projectId="akka-cluster-typed" }
+
+## Dependency
+
+To use Distributed Publish Subscribe add the following dependency in your project:
+
+@@dependency[sbt,Maven,Gradle] {
+  group=com.typesafe.akka
+  artifact=akka-cluster-typed_$scala.binary_version$
+  version=$akka.version$
+}
+
+## Introduction
```

Comment:
I think this documentation looks too ambitious, given that we want to provide a new API for Typed, and that will probably be rather different. Probably based on the Receptionist as in Johan's spike https://github.com/akka/akka/pull/26340

I think we should keep the descriptions here to a minimum and refer to the Classic docs and link to the issue about new API https://github.com/akka/akka/issues/26338.

Including the example here would still be good.

The main point about this page is that the API for Typed is not ready yet, but the Classic APIs can be used with the `toClassic` adapters. Could also link to coexisting.md.

Reply:
Updated to minimal.